unlock-app/src/actions/error.js

```diff
@@ -1,6 +1,20 @@
 export const SET_ERROR = 'SET_ERROR'
+export const RESET_ERROR = 'RESET_ERROR'
+export const WEB3_ERROR = 'WEB3_ERROR'
 
-export const setError = error => ({
-  type: SET_ERROR,
+export const setError = (error, type = 'SET_ERROR') => ({
```

Comment:
Hum... I think I must not have been describing what I wanted very well. This is overly complicated. we do not need 2 values. Just one.